scripts/globals/weaponskills.lua

```diff
@@ -481,35 +491,55 @@ xi.weaponskills.calculateRawWSDmg = function(attacker, target, wsID, tp, action,
         if calcParams.sneakApplicable then
             dmg = dmg + attacker:getStat(xi.mod.DEX)
         end
+
         if calcParams.trickApplicable then
             dmg = dmg + attacker:getStat(xi.mod.AGI)
         end
     end
 
     -- Calculate critrates
-    local critrate = calculateDEXvsAGICritRate(attacker, target)
+    local critrate = 0.05 + calculateDEXvsAGICritRate(attacker, target) -- 5% constant crit floor + dDEX
+    -- mainhand and offhand weapon crit mods aren't included with the regular CRITHITRATE mod
+    local mainSlotCritBonus = 0
+    local subSlotCritBonus = 0
+    if attacker:getEquippedItem(xi.slot.MAIN) ~= nil then
```

Comment:
Just an FYI you do a `~=nil` check here for main hand but then don't do it for the off hand if check below. Just pointing it out incase it wasn't intentional.

Reply:
Yeah, I should include it there. Nice catch.